packages/frontend/src/pages/settings/drive.vue

```diff
@@ -106,6 +105,14 @@ const meterStyle = computed(() => {
 
 const keepOriginalUploading = computed(defaultStore.makeGetterSetter('keepOriginalUploading'));
 const imageCompressionMode = computed(defaultStore.makeGetterSetter('imageCompressionMode'));
+const imageResize = ref(!!imageCompressionMode.value?.startsWith('resize'));
+const imageCompressionLossy = ref(!!imageCompressionMode.value?.endsWith('CompressLossy'));
```

Comment:
```suggestion
const imageResize = ref(imageCompressionMode.value?.startsWith('resize') ?? true);
const imageCompressionLossy = ref(imageCompressionMode.value?.endsWith('CompressLossy') ?? false);
```

デフォルト値をわかりやすくしたいです。
またもともとのデフォルトが`resizeCompress`なのでnullのときには`imageResize`が`true`になるようにしたいです。

Reply:
こう書きましたがそもそもimageCompressionModeがnullableにしてるのなんでだ...?nullableでなくて良さそうなのでそのままで良さそうですねすみません。

Reply:
独自実装であり元々存在しないデータだったからとか…? デフォルト値代入するようになってるとかなら謎ですが

Reply:
デフォルト値が入ってるはずで、何故か明示的にnullを許容するようになってるのですよね

Reply:
そうなんですよね... nullable が気になってこのような実装になりました。
ではここはこのままでいきます。

Reply:
見た感じ`reactionAcceptance`からコピペしてたっぽいですね... reactionAcceptanceはnullに意味がちゃんとあるのであまり考えずにコピペしてそう...すみません